browser/css/jsdialogs.css

```diff
@@ -129,14 +135,14 @@ div#autoFillPreviewTooltip .lokdialog.ui-dialog-content.ui-widget-content {
 
 #SortKeyWindow {
 	/* As used in .jsdialog-container .ui-dialog-content 90 % of the viewport
-	   minus 352px:
+		 minus 352px:
 		- 34px the dialog's header
 		- 112px the next sibling
 		- 50px the dialog's buttons
 		- 80px status bar and calc sheet bar
 		- 38px formulabar
 		- 38px document-header
-   */
```

Comment:
please discard this comment formatting changes

Reply:
I had formated the file to use Tabs and these were using Spaces